editor/modes/visual-editor/block.js

```diff
@@ -229,7 +241,26 @@ class VisualEditorBlock extends wp.element.Component {
 						</div>
 					</CSSTransitionGroup>
 				}
-				<div onKeyPress={ this.maybeStartTyping }>
+				{ this.props.isFirstSelected && (
```

Comment:
I'm ok keeping it to avoid conflicting with the selector. but Maybe I'd rename the selector `isFirstSelectedBlock`